package.json

```diff
@@ -12,13 +12,16 @@
     "@react-three/drei": "^9.106.1",
     "@react-three/fiber": "^8.16.8",
     "@use-gesture/react": "^10.3.1",
+    "classnames": "^2.5.1",
```

Comment:
you also don't use this package anywhere I believe

Reply:
![image](https://github.com/user-attachments/assets/45daa248-2946-4a04-ba8a-a3d0ca877501)